src/fixtures/media.ts

```diff
@@ -0,0 +1,70 @@
+import { MediaFile } from "@root/utils/media-utils"
+
+export const MEDIA_FILE_NAME = "test file"
+export const MEDIA_SITE_NAME = "site"
+export const MEDIA_DIRECTORY_NAME = "dir"
+export const MEDIA_FILE_SHA = "sha"
+
+export const mediaDir: MediaFile = {
```

Comment:
curious: why not SCREAMING_SNAKE_CASE? see the google style guide [here](https://google.github.io/styleguide/tsguide.html#identifiers-constants)

Reply:
Changed in https://github.com/isomerpages/isomercms-backend/pull/741/commits/625680525dcc9f0aa9bea12c5f8fa3a7cd172070!